doc/00-installation.md

```diff
@@ -0,0 +1,64 @@
+# Installation
+
+As `image-builder-cli` is still in development its packages are not yet included in any distribution. There are however multiple ways that you can already use it. After you have `image-builder-cli` installed take a look at its [usage](./01-usage.md).
```

Comment:
(nitpick) not a blocker and I understand this is a moving target so we could merge without but this is no longer true, we are in fedora now!